src/test/java/uk/gov/hmcts/reform/sscs/personalisation/PersonalisationTest.java

```diff
@@ -637,6 +641,40 @@ public void shouldPopulateRegionalProcessingCenterFromCcdCaseIfItsPresent() {
         assertEquals(POSTCODE, result.get(POSTCODE_LITERAL));
     }
 
+    @Test
+    public void shouldPopulateRegionalProcessingCenterWhenNotGoingToGaps() {
```

Comment:
Can we update this test name to something more specific to what is going on? You're not populating the Regional Processing Center, you're populating the evidence processing address. Plus the case is still going to gaps, just not until createdInGapsFrom is set to 'ready to list'